provider/kubernetes_test.go

```diff
@@ -1524,11 +1524,80 @@ func TestServiceAnnotations(t *testing.T) {
 	}
 }
 
+func TestGetServiceErrors(t *testing.T) {
+	ingresses := []*v1beta1.Ingress{
+		{
+			ObjectMeta: v1.ObjectMeta{
+				Namespace: "testing",
+				Annotations: map[string]string{
+					"traefik.frontend.passHostHeader": "true",
```

Comment:
Do we need this annotation for the tests at hand?

I'm always in favor of keeping test fixtures as minimized as possible.